src/components/PullRequestList/index.tsx

```diff
@@ -36,8 +36,9 @@ const PullRequestList: FC<PullRequestListProps> = ({ prType }) => {
   const [noData, setNoData] = useState(false);
   const [page, setPage] = useState(1);
   const [isBottom, setIsBottom] = useState(false);
+  const size = Math.floor((global.innerHeight / 147) * 3);
```

Comment:
Normally `global` is for the node's runtime and `window` is for browsers. I am not sure how it's working here, I'll try to figure that out.
But for now let's change `global` to `globalThis` so there is no confusion of this sort.

> Recently, globalThis was added to the language, as a standardized name for a global object, that should be supported across all environments. It’s supported in all major browsers.
Refer - https://javascript.info/global-object